specification/configuration.md

```diff
@@ -250,12 +250,51 @@ are required.
 
 In addition to environment variables, other ways of defining configuration also exist:
 
-- [Java System
-  Properties](https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html):
-  These properties MUST match the environment variables converting to lower
-  case and replacing underscores with hyphens or periods. For example:
-  system property `splunk.trace-response-header.enabled` is equivalent to environment
-  variable `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED`.
+#### File based configuration
+
+OpenTelemetry's [declarative configuration](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/README.md#declarative-configuration) SHOULD be supported via [`OTEL_EXPERIMENTAL_CONFIG_FILE`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk.md#via-otel_experimental_config_file) environment variable.
+
+In addition, Splunk specific configuration MUST have their own root-level configuration block named `splunk`.
+
+The following is an example covering the common configuration values:
+
+```yaml
+splunk:
+  # Common values across language distributions.
+  realm: "eu0"
+  accessToken: "${SPLUNK_ACCESS_TOKEN}"
```

Comment:
We have discussed it already. This should be configurable by standard OTel properties. Need to prepare only snippets.

Reply:
If the file is intended to be edited manually, then it's subpar UX. You'd have to modify all of the signal exporters, as a an end user I'd be asking "Couldn't you just make it editable in one location?".

For us it's definitely simpler like that.